### DISCUSSION TITLE
吃什么能用了吗？还是我有问题？

### DISCUSSION THREAD

**Discussion by ABT98421** [Arondight/Adachi-BOT#746] [Q&A] [answered]
一发吃什么就会闪退
![image](https://user-images.githubusercontent.com/78371203/159477884-d535c233-d75e-4cf1-947e-2dab6dd72af3.png)

**Comment by Arondight** [comment, marked as answer]
这个插件曾经重写过一次，配置文件的格式有改变

**Comment by ABT98421** [comment]
那么问题又来了，现在是怎么更新的。好像自从某天开始我就没更新成功过
![image](https://user-images.githubusercontent.com/78371203/159479513-448bc118-5849-42dd-900d-3956d6f9fb9b.png)

**Reply by mark9804** [reply]
你之前有没有自己维护过一份菜单，如果有的话需要删掉，然后重启

**Reply by ABT98421** [reply]
原来如此，好了
